bindings/pyroot/pythonizations/python/ROOT/pythonization/_tfile.py

```diff
@@ -8,10 +8,76 @@
 # For the list of contributors see $ROOTSYS/README/CREDITS.                    #
 ################################################################################
 
-# TFile inherits from
-# - TDirectory the pythonized attr syntax (__getattr__) and WriteObject method.
-# - TDirectoryFile the pythonized Get method (pythonized only in Python)
-# what is left to add is the pythonization of TFile::Open.
+r'''
+/**
+\class TFile
+\brief \parblock \endparblock
+\htmlonly
+<div class="pyrootbox">
+\endhtmlonly
+## PyROOT
+
+From Python, it is possible to inspect the content of a TFile object
```

Comment:
Same holds true here, such as pointed out for the `TFileDirectory`. If possible, I would reduce the redundancy.